middle.mkd

```diff
@@ -280,20 +280,23 @@ The default reference system for all GeoJSON coordinates SHALL be
 a geographic coordinate reference system, using the [WGS84] datum, and
 with longitude and latitude units of decimal degrees. This coordinate
 reference system is equivalent to the OGC's
-"http://www.opengis.net/def/crs/OGC/1.3/CRS84" [OGCURL]. An OPTIONAL
-third position element SHALL be the height in meters above the WGS 84
-reference ellipsoid. For widest interoperability, GeoJSON data SHOULD
-use this default coordinate reference system.
+"http://www.opengis.net/def/crs/OGC/1.3/CRS84" [OGCURL]. To maximize
+interoperability, GeoJSON data SHOULD use this default coordinate
+reference system.  An OPTIONAL third position element SHALL be the
+height in meters above the WGS 84 reference ellipsoid. In the absence of
+elevation values, applications sensitive to height or depth SHOULD
+interpret positions as being at local ground or sea level.
 
 Other coordinate reference systems, including ones described by CRS
 objects of the kind defined in [GJ2008] are NOT RECOMMENDED. GeoJSON
 processing software SHALL NOT be expected to have access to coordinate
 reference systems databases.  Applications requiring CRS other than the
 default MUST assume all responsibility for reference system and
-coordinate accuracy. Furthermore, GeoJSON coordinates MUST NOT under any
-circumstances use latitude, longitude order. See Section 8,
-Interoperability Considerations, for guidance in processing GeoJSON
-documents that do contain such a CRS object.
+coordinate accuracy and interpretation of missing elevation values.
```

Comment:
Should this be "reference system, coordinate accuracy, and interpretation of missing elevation values"?